layers/+spacemacs/spacemacs-layouts/packages.el

```diff
@@ -112,7 +112,7 @@
       ;; re-activate during `dotspacemacs/sync-configuration-layers' - see
       ;; issues #5925 and #3875)
       (unless (bound-and-true-p persp-mode)
-        (persp-mode))
+        (spacemacs/defer-until-after-user-config #'persp-mode))
```

Comment:
I'm not sure how the deferring works here, but won't it be more robust to defer the `bound-and-true-p` guard as well?